runtime/flang/bcopys.c

```diff
@@ -26,7 +26,7 @@ __fort_bcopysl(char *to, char *fr, size_t cnt, size_t tostr, size_t frstr,
   size_t i, j;
   unsigned long n;
 #if !defined(WIN64)
-  long k;
+  size_t k;
 #else
   long long k;
```

Comment:
I don't like leaving it like that. Later on, `k` is used for comparison with `len` which is of unsigned type `size_t`. Imo, `k` should be always of type `size_t` regardless the OS targeted.

Reply:
You're right. `k` should always be of type `size_t`.